packages/material-ui/src/Chip/Chip.js

```diff
@@ -87,7 +87,7 @@ export const styles = (theme) => {
       userSelect: 'none',
       WebkitTapHighlightColor: 'transparent',
       cursor: 'pointer',
-      '&:hover, &:focus': {
+      '&$focusVisible, &:hover': {
```

Comment:
It would have been better to reduce the diff

```suggestion
      '&:hover, &$focusVisible': {
```